src/Microsoft.NetCore.Analyzers/Core/Security/DoNotDisableCertificateValidation.cs

```diff
@@ -0,0 +1,180 @@
+                                            Rule,
+                                            kindOfTargetFunction.ToString()));
+                                }
+                            }
+                        },
+                        OperationKind.DelegateCreation);
+                });
+        }
+
+        /// <summary>
+        /// Gets all valid members of the block operation body, excluding the VB implicit statements.
+        /// </summary>
+        /// <param name="operations">All the descendants of the IBlockOperation of target method.</param>
+        private static ImmutableArray<IOperation> FilterImplicitOperations(ImmutableArray<IOperation> operations)
+        {
+            if (operations.IsDefaultOrEmpty)
+            {
+                return operations;
+            }
+
+            if (operations.Length > 2 && operations[0].Language == LanguageNames.VisualBasic)
+            {
+                var lastOperation = operations[operations.Length - 1];
+                var secondLastOperation = operations[operations.Length - 2];
+                var thirdLastOperation = operations[operations.Length - 3];
+
+                if (lastOperation.Kind == OperationKind.LocalReference && lastOperation.IsImplicit &&
+                    secondLastOperation.Kind == OperationKind.Return && secondLastOperation.IsImplicit &&
+                    thirdLastOperation.Kind == OperationKind.Labeled &&
+                    ((ILabeledOperation)thirdLastOperation).Label.Name == "exit" &&
+                    thirdLastOperation.IsImplicit)
+                {
+                    var builder = ImmutableArray.CreateBuilder<IOperation>();
+                    builder.AddRange(operations, operations.Length - 3);
+                    return builder.ToImmutable();
+                }
+                else
+                {
+                    return operations;
+                }
+            }
+            else
+            {
+                return operations;
+            }
+        }
+
+        /// <summary>
+        /// Find every IReturnOperation in the method and get the value of return statement to determine if the method always return true.
+        /// </summary>
+        /// <param name="operation">A method body in the form of explicit IOperations</param>
+        private static bool AlwaysReturnTrue(IEnumerable<IOperation> operations)
+        {
+            var result = true;
+            var countOfReturnStatement = 0;
+            
+            foreach (var descendant in operations)
+            {
+                if (descendant.Kind == OperationKind.Return)
+                {
+                    var returnOperation = (IReturnOperation)descendant;
+                    var constantValue = returnOperation.ReturnedValue.ConstantValue;
+
+                    countOfReturnStatement++;
+
+                    // If the target method invokes another function which is from local or 3rd assembly,
```

Comment:
I would write this comment as - `Check if the value being returned is a compile time constant 'true'`